service/src/main/java/uk/gov/hmcts/reform/fpl/service/OrderValidationService.java

```diff
@@ -3,23 +3,28 @@
 import org.springframework.stereotype.Service;
 import uk.gov.hmcts.reform.fpl.model.CaseData;
 
+import java.util.ArrayList;
 import java.util.List;
 
-import static java.util.Collections.emptyList;
-import static java.util.Collections.singletonList;
+import static java.util.Objects.isNull;
 import static org.springframework.util.CollectionUtils.isEmpty;
 import static uk.gov.hmcts.reform.fpl.enums.OrderStatus.SEALED;
 
 @Service
 public class OrderValidationService {
     public List<String> validate(final CaseData caseData) {
+        List<String> validationErrors = new ArrayList<>();
         if (SEALED == caseData.getStandardDirectionOrder().getOrderStatus()
             && isEmpty(caseData.getHearingDetails())) {
-            return singletonList(
-                "This standard directions order does not have a hearing associated with it. "
-                    + "Please enter a hearing date and resubmit the SDO");
+            validationErrors.add("This standard directions order does not have a hearing associated with it. "
+                + "Please enter a hearing date and resubmit the SDO");
         }
 
-        return emptyList();
+        if (SEALED == caseData.getStandardDirectionOrder().getOrderStatus()
+            && isNull(caseData.getAllocatedJudge())) {
+            validationErrors.add("Enter allocated judge");
```

Comment:
I know it's the AC but compared with our other error message we display for SDO, this one is a lot less informative - perhaps could discuss with BA? not a problem though, just my thoughts

Reply:
Yes I mentioned that to Collette and she said she would get back to me on it!

Reply:
Great 👍

Reply:
I've changed these now based on what colette suggested